client/log_client_test.go

```diff
@@ -227,6 +229,23 @@ func TestWaitForInclusion(t *testing.T) {
 	}
 }
 
+func isDeadlineExceeded(err error) bool {
+	if err == nil {
+		return false
+	}
+	if status.Code(err) == codes.DeadlineExceeded {
+		return true
+	}
+	// TODO(pavelkalinnikov): Sometimes Trillian gRPC returns codes.Unknown for a
```

Comment:
Is there an issue to fix this? Can you create one if not.

Reply:
Done: #2328